src/dist/replication/meta_server/meta_service.h

```diff
@@ -218,6 +218,10 @@ class meta_service : public serverlet<meta_service>
     //  -1. meta isn't leader, and rpc-msg can't forward to others
     // if return -1 and `forward_address' != nullptr, then return leader by `forward_address'.
     int check_leader(dsn::message_ex *req, dsn::rpc_address *forward_address);
+    template <typename TRpcHolder>
+    int check_leader(TRpcHolder rpc, /*out*/ rpc_address *forward_address);
+    template <typename TRpcHolder>
+    bool check_status(TRpcHolder rpc, /*out*/ rpc_address *forward_address = nullptr);
```

Comment:
I checked all the callers of `check_status`. None is using the parameter `forward_address `. It's useless I think.
```suggestion
    bool check_status(TRpcHolder rpc);
```

Reply:
I will use it in the later pull request, which needs the forward_address.